site/blog/2022-10-27-clean-your-aws-account.mdx

```diff
@@ -0,0 +1,108 @@
+---
+slug: clean-your-aws-account
+title: Clean your AWS account just by clicking a button
+authors: [yrobla]
+---
+
+# Clean your AWS account by simply clicking a button
+
+One of the main complexities of working with AWS is the amount of resources that can be created, either by manual or automated workflows. The chained resources created and their dependencies can quickly get out of control, and it could be a nightmare to clean them up.
+
+Having unknown resources running will cause you to incur on extra costs, or could cause unexpected conflicts. Cleaning those manually can be a hard task... manually visiting each AWS service page in the console, investigating the dependencies, manually cleaning... it is a time-consuming, error-prone task.
+
+To solve this problem you can use our IaSQL engine. By simply clicking a button, our engine will get an snapshot of the status of your cloud, showing you all the resources that you have created over the time, on multiple regions.
+There may be some resources that you are aware of and you need to keep, but also some leftovers or unknown resources created by some automated process.
+
+**Want to give a try? Join our service for free and start taking control of your cloud.**
```

Comment:
I would move this to the end and maybe something less sales-y like

Connect your AWS account to IaSQL for free using the [dashboard](app.iasql.com). To read more about AWS account management check our documentation guide https://iasql.com/docs/aws/

Reply:
what part to do you want me to move? the button one?